frontend/packages/metal3-plugin/src/components/host.tsx

```diff
@@ -99,9 +98,7 @@ const HostsTableRow: React.FC<HostsTableRowProps> = ({
       <TableData className={tableColumnClasses[1]}>
         <BaremetalHostStatus status={status} />
       </TableData>
-      <TableData className={tableColumnClasses[2]}>
-        <NodeCell nodeName={nodeName} namespace={namespace} />
-      </TableData>
+      <TableData className={tableColumnClasses[2]} />
```

Comment:
This is incorrect probably due to the rebase. This should not be changed.

Reply:
It's what linter suggested.  I did this to preserve what I understood was the order in which the cells of the table were rendered.  What would be a better way to do this?  Modify the index used to reflect one of the datas no longer being present (e.g. decrement the number by 1 for data that follows this removal)

Reply:
Originally this `<TableData>` rendered a `MachineCell`, now it correctly renders `NodeCell`. This PR should not do any changes to this part of the code.